src/cntk-train/src/main/scala/CommandBuilders.scala

```diff
@@ -181,8 +184,9 @@ class MPICommandBuilder(log: Logger,
         Seq("scp", "-i", identity, "-r", "-o", "StrictHostKeyChecking=no",
           fileInputPath, s"$gpuUser:$fileInputPath"))
     }
+    val runMPI = "''" + s"$exportClasspath $command $mpiArgs $cntkArgs" + "''"
     // Run the mpi command
-    printOutput(Seq("ssh", "-i", identity, gpuUser, command, mpiArgs, cntkArgs, "parallelTrain=true"))
+    printOutput(Seq("ssh", "-i", identity, gpuUser, runMPI))
```

Comment:
Just want to make sure we are not losing "parallelTrain=true"

Reply:
it didn't seem to make a difference for me, I believe it's already in the override config file